frontend/public/components/pod.tsx

```diff
@@ -341,15 +341,12 @@ const PodGraphs = requirePrometheus(({ pod }) => (
   </>
 ));
 
-export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => <Status status={podPhase(pod)} />;
-
 export const PodDetailsList: React.FC<PodDetailsListProps> = ({ pod }) => {
   return (
     <dl className="co-m-pane__details">
-      <dt>Status</dt>
-      <dd>
-        <PodStatus pod={pod} />
-      </dd>
+      <DetailsItem label="Status" obj={pod} path="status.phase">
```

Comment:
I'm on the fence about this change because we're not showing the straight `status.phase` value, so it could be somewhat misleading. In some cases, we're showing container state. We intentionally didn't use the API doc here for that reason.

Reply:
@spadgett I see that the `podPhase` is returning different phase types as is the enumerate of the `pod.status.phase` field.
Now I see that it might be confusing for the user since we are not using the same types. That said, shall I close the PR together with the BZ ?

Reply:
I think we can close this as not a bug. The API doc might be misleading in this case because it's not exactly what we're showing.